src/Header.tsx

```diff
@@ -1,146 +1,117 @@
-              <Navbar.Text>Signed in as {username}</Navbar.Text>
-              <Nav.Link>
-                <button
-                  onClick={() => {
-                    AccountService.logout();
-                    window.location.href = '/login?left';
-                  }}
-                  style={{
-                    background: 'none',
-                    outline: 'none',
-                    border: 'none',
-                  }}
-                >
-                  Logout
-                </button>
-              </Nav.Link>
-            </>
-          ) : (
-            <Nav.Link href='/login'>Private Switzerland Login</Nav.Link>
-          )}
-          <div
-            style={{
-              display: 'flex',
-              justifyContent: 'center',
-              alignItems: 'center',
-              cursor: 'pointer',
-              padding: '0 0.5rem 0 0.5rem',
-            }}
-            onClick={() => window.open('https://github.com/cevo-public/cov-spectrum-website', '_blank')}
-          >
-            <AiOutlineGithub size={'1.5em'} />
+    <>
+      <nav className='flex h-full content-center shadow-md z-50 bg-white '>
+        <div className='w-full mx-auto px-2 md:px-0 flex content-center'>
+          <div className='md:mx-4 w-full justify-between'>
+            <div className='w-full h-full flex justify-center md:justify-between items-center'>
+              <div id='logo-and-search' className='flex h-full md:flex-row flex-column justify-center'>
+                <div id='logo-and-gsid' className='flex flex-column items-center justify-center md:pr-4'>
+                  <div>{Logo}</div>
+                  <div className='text-xs flex flex-row justify-between space-x-1'>
+                    <div className='self-end text-gray-500 text-sm'>Enabled by data from </div>{' '}
+                    <ExternalLink url='https://gisaid.org/'>
+                      <img src='/img/gisaid.png' alt='GISAID' style={{ height: '20px' }} />{' '}
+                    </ExternalLink>
+                  </div>
+                </div>
+                <div className='flex items-center z-20 mt-2 md:mt-0'>
+                  <HeaderCountrySelect />
+                  <div id='date-range-wrapper'>
+                    <HeaderDateRangeSelect />
+                  </div>
+                  <div className='hidden xl:block'>
+                    <HeaderSamplingStrategySelect />
+                  </div>
+                </div>
+              </div>
+              <div id='right-nav-buttons' className='hidden md:block items-center justify-center'>
+                <div className='ml-1 flex items-center'>
+                  <a
+                    className={`${getButtonClasses('/acknowledgements')} hidden lg:block`}
+                    href='/acknowledgements'
+                  >
+                    Acknowledgements
+                  </a>
+                  <a className={getButtonClasses('/about')} href='/about'>
+                    About
```

Comment:
It's a bit sad that the about page is not accessible on mobile, since that part of the menu is hidden.

Reply:
@chaoran-chen Could this be a problem with our GISAID contract, since acknowledgements is also hidden?

Reply:
Both the about and acknowledgements page should be made accessible on mobile.